phpstan-baseline.neon

```diff
@@ -0,0 +1,77 @@
+parameters:
+	ignoreErrors:
```

Comment:
do we need baseline at all? most of this errors are easy fix (like dataproviders). Others will be ignored forever until we bring phpstan to some key dependencies

Reply:
I think we do. The errors in baseline are not worth fixing I think. And it'll take a lot of time to fix them.

Reply:
Yeah, sounds reasonable. I'll do it by myself later and drop baseline when I fix everything